doc/changelog.rst

```diff
@@ -6,6 +6,11 @@ Changes in Version 4.4
 
 - Added support for passing a list containing (key, direction) pairs
   or keys to :meth:`~pymongo.collection.Collection.create_index`.
+- **BETA** Added support for range queries on client side field level encrypted collections.
+- pymongocrypt 1.5.0 or later is now required for client side field level
+  encryption support.
+- Improved support for Pyright to improve type support for IDEs like Intellisense.
```

Comment:
Is Intellisense an IDE? Maybe "IDEs like Visual Studio and VS Code."

Reply:
You're right, I got it confused. Fixed.